Dockerfile

```diff
@@ -9,6 +9,12 @@ COPY package.json package-lock.json ./
 # Install Node Packages
 RUN npm install
 
+# Copying self-signed certs
+COPY certs/ certs/
```

Comment:
For a fresh install, this folder will be empty save for the certgen script.

On another hand, for an install that had already generated certs, this will result in copying over (potentially old) certs.

Wouldn't it be better to just `mkdir certs` and `COPY` the script separately to avoid this?

I'm not sure that the certgen script is passing flags to openssl to overwrite files if they exist, but I think we should just avoid the potential issue altogether.

My suggestion would be something like:

```Dockerfile
RUN mkdir certs/
COPY certs/certgen.sh certs/certgen.sh
RUN cd certs && ./certgen.sh
```

Reply:
Just tested and openssl will overwrite. Nevertheless, I could do something like that. `COPY certs/certgen.sh certs/certgen.sh` should work without the mkdir.

Reply:
Does that create the folder? I was not sure of that.

If so, sounds fine, and avoids copying unnecessary certs. It may just be annoying to have to go and accept the cert everytime the container is built, but since this is after the install, it should only run again if package*.json change so it shouldn't be _too_ often I guess.